tests/models/test_cql.py

```diff
@@ -0,0 +1,37 @@
+# pylint: disable-all
+from datetime import datetime
+
+import pytest
+from pyspark.sql import DataFrame
+
+from replay.constants import LOG_SCHEMA
+from replay.models import CQL
+from tests.utils import spark
+
+
+@pytest.fixture
+def log(spark):
```

Comment:
Hi! Thanks for collaboration! I will add comments on the model testing.

We already have some fixtures in tests/utils, e.g. log, log2, long_log_with_features to use in tests.  Please reuse them here. 

We also have some tests in tests/models/test_all_models. Consider adding your model to some of those tests, e.g. your may test your model works as expected in case of new users/items appearance on inference (test_predict_cold_and_new_filter_out and test_predict_new_users/test_predict_cold_users if applicable for your model).  

If you added model saving/loading functionality, please test it in tests/models/test_save_load_models.py.

Reply:
Thank you for point out to these test files. Added our model to the list of input parameters for applicable tests.